broker/cloud_functions/store_BigQuery/main.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""This module stores alert data in BigQuery tables."""
+
+import base64
+from google.cloud import logging
```

Comment:
I'm not sure how to fix this Codacy error, and the following one. Is there a requirements file/list somewhere that needs to be updated? If so, noting that there are more of these in #71.

Reply:
Hmmm. This is a mypy issue. I'll take a look at it and let you know what I find.